landing.css

```diff
@@ -28,70 +28,51 @@ a {
     margin-top: 2rem;
 }
 
-.category-title {
-    text-align: left;
-    font-size: 2.5rem;
+.category-subtitle {
+    text-align: center;
+    font-size: 3rem;
     color: var(--content-color);
-    margin-top: 5rem;
-    margin-left: 8rem;
+    margin-top: 10rem;
 }
 
-.quiz-item {
+.category-selector {
+    margin-top: 1rem;
     display: flex;
     align-items: center;
     justify-content: center;
+    gap: 2rem;
     flex-wrap: wrap;
-    margin-top: 5px;
-}
-
-.quiz-img {
-    height: 20rem;
-    width: auto;
-}
-
-.quiz-item .card-img {
-    object-fit: fill;
-}
-
-.quiz-item .card-container.product-card {
-    width: 35rem;
-    height: 35rem;
-    background-color: var(--for-quiz);
 }
 
-.quiz-item .card-title,
-.quiz-item .card-subtitle,
-.quiz-item .card-description {
+.category-title {
+    text-align: center;
+    font-size: 2.5rem;
     color: var(--content-color);
-    margin-right: 2rem;
-    padding: 5px 2rem;
-}
-
-.quiz-item .card-title {
-    font-size: 1.5rem;
-    font-weight: 500;
-    /* margin-top: auto; */
+    margin-top: 2rem;
+    background-color: var(--for-quiz);
```

Comment:
just a suggestion on this variable name, it would make more sense if it would be something like --quiz-color, rest looks good.

Reply:
thanks for pointing it out i will think for a new name